GOVERNANCE.md

```diff
@@ -1,21 +1,37 @@
 # AMP Project open source governance
 
-The AMP Project's chief and primary concern is with the development of AMP HTML, an open-source runtime shared by many producers and consumers of (mostly) static web content.  Its governance model thus reflects only the need to steer engineering direction, and not any other activities, which would be out of scope.
+This document describes the governance model for the AMP open source project, and in particular the [AMP HTML GitHub project](https://github.com/ampproject/amphtml).
 
 Our governance model is as follows:
 
-* There is a single Tech Lead, who will have the final say on all decisions regarding technical direction.
-* The Tech Lead directs the Core Committers, whose members include the Tech Lead and those who have been appointed by the Tech Lead as Core Committers.
+* Changes to the AMP repository have to be approved by the `OWNERS` of the code. Owners are specified by directory via the [OWNERS.yaml](https://github.com/ampproject/amphtml/search?utf8=%E2%9C%93&q=filename%3AOWNERS.yaml&type=Code) file.
+* The creator(s) of a particular component, extensions, plugin or sub system should typically name themselves as owners and they are free to delegate ownership to other GitHub users or teams of the [AMP Project GitHub Project](https://github.com/ampproject).
+* Just to rephrase: Ownership of a sub system is **given out liberally** to whoever seems most knowledgeable about a piece of code. Typically the creator should designate themselves.
+* Owners of higher level directories automatically have ownership (approval rights) for sub directories.
+* These higher level owners would, for example, approve the creation of a new component.
+* On top of owners approval, each pull request additionally requires review and approval from an AMP Project core committer.
```

Comment:
May want to clarify that if an OWNER is also a core committer approval from that person is sufficient (i.e. you don't need two people to approve every PR).